Source/Scene/Cesium3DTile.js

```diff
@@ -553,11 +558,18 @@ define([
      */
     Cesium3DTile.prototype.createBoundingVolume = function(boundingVolumeHeader, transform, result) {
         var center;
+        var hasTileTransform = !Matrix4.equals(this.computedTransform, Matrix4.IDENTITY);
         if (defined(boundingVolumeHeader.box)) {
             var box = boundingVolumeHeader.box;
             center = Cartesian3.fromElements(box[0], box[1], box[2], scratchCenter);
             var halfAxes = Matrix3.fromArray(box, 3, scratchHalfAxes);
 
+            if (hasTileTransform) {
+                // Transform from y-up to z-up
+                Matrix3.multiply(yUpToZUpMatrix3, halfAxes, halfAxes);
+                Matrix4.multiplyByPoint(yUpToZUpMatrix4, center, center);
+            }
```

Comment:
The spec now defines local bounding volumes as y-up. So I do the y-up to z-up conversion at runtime. I was also wondering about transforms in local space...

Say you have a root and a child. The root has a transform that goes from local to wgs84 and the child has a transform that translates it upward by 10 meters. Intuitively the child's transform should be a translation of (0.0, 10.0, 0.0) to match the y-up convention of local-space. However the translation needed by Cesium is (0.0, 0.0, 10.0). So the transforms are z-up and bounding volumes are y-up. My change below with transforming bounding volumes from y-up to z-up may also be error prone since sometimes the volume is already in wgs84 space, and sometimes it's in local space, and it's hard to tell. I just make the decision by if there's a tile transform, but that doesn't seem great.

The main issue is how do we make the distinction that a transform/bounding volume is in local space vs wgs84 space? Should all transforms be y-up including the wgs84 transform? Or should we instead define z-up as the convention?